plugins/generator-1.17.1/forge-1.17.1/jsontriggers/block_placed.json.ftl

```diff
@@ -1,8 +1,6 @@
 {
   "trigger": "minecraft:placed_block",
   "conditions": {
-    "item": {
-      "item": "${input$block}"
-    }
+    "block": "${input$block}"

```

Comment:
I think we can use item, but it needs to be string, not object. Using item will allow this trigger to also work with items that turn into blocks. Ref: https://minecraft.fandom.com/wiki/Advancement/JSON_format#minecraft:placed_block

Reply:
Did not test this though, just read the wiki

Reply:
I remember to have tested with item before using block. It will work.